index-handlers/src/main/java/no/sikt/nva/nvi/index/model/Publication.java

```diff
@@ -0,0 +1,87 @@
+package no.sikt.nva.nvi.index.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import java.util.Objects;
+import nva.commons.core.JacocoGenerated;
+
+public final class Publication {
```

Comment:
Can this be called something else so its not condused with Publication from the datamodel?

Reply:
Renamed to PublicationDetails